.github/workflows/test.yml

```diff
@@ -319,31 +318,29 @@ jobs:
         with:
           name: report-integration.xml
 
-      # - name: Download pre-upgrade test Report
-      #   uses: actions/download-artifact@v4
-      #   with:
-      #     name: report-pre-upgrade.xml
+      - name: Download pre-upgrade test Report
+        uses: actions/download-artifact@v4
+        with:
+          name: report-pre-upgrade.xml
 
-      # - name: Download post-upgrade test Report
-      #   uses: actions/download-artifact@v4
-      #   with:
-      #     name: report-post-upgrade.xml
+      - name: Download post-upgrade test Report
+        uses: actions/download-artifact@v4
+        with:
+          name: report-post-upgrade.xml
 
-      # - name: Download upgraded integration test Report
-      #   uses: actions/download-artifact@v4
-      #   with:
-      #     name: report-upgraded-integration.xml
+      - name: Download upgraded integration test Report
+        uses: actions/download-artifact@v4
+        with:
+          name: report-upgraded-integration.xml
 
       # - name: Download pricing proposal test Report
       #   uses: actions/download-artifact@v4
       #   with:
       #     name: report-pricing-proposal.xml
-
       # - name: Download pricing change test Report
       #   uses: actions/download-artifact@v4
       #   with:
       #     name: report-pricing-change.xml
```

Comment:
Both _should_ be enabled, as long as pricing parameter change proposal is actively tested as well.